apps/team-awesome-web/src/components/__test__/testServices.test.tsx

```diff
@@ -0,0 +1,40 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+
+import { getTimeEntries } from "../../services/time-entries/getTimeEntries";
+
+const mockedTimeEntries = [
+  {
+    activity: "developmentwork",
+    client: "Port of Rotterdam",
+    startTimestamp: "2021-10-08T07:01:00.000Z",
+    stopTimestamp: "2021-10-08T15:00:00.000Z",
+    id: 0,
+  },
+];
+
+const unmockedFetch = global.fetch;
+
+beforeEach(() => {
+  global.fetch = jest.fn(() =>
+    Promise.resolve({
+      json: () => Promise.resolve(mockedTimeEntries),
+    }),
+  ) as jest.Mock;
+});
+
+afterEach(() => {
+  global.fetch = unmockedFetch;
+});
+
+test("if time entries are fetched from the server", async () => {
+  const response = await getTimeEntries();
```

Comment:
This is not the response from the fetch. These are the actual time entries, or perhaps undefined and/or an error